code/datums/antagonists/ninja.dm

```diff
@@ -58,7 +58,7 @@
 				var/is_bad_guy = possible_targets[M]
 				possible_targets.Cut(index,index+1)
 
-				if(is_bad_guy ^ helping_station)			//kill (good-ninja + bad-guy or bad-ninja + good-guy)
+				if(is_bad_guy && !helping_station)			//kill (good-ninja + bad-guy or bad-ninja + good-guy)
```

Comment:
Revert these.

Reply:
I wish I read the comment first, thanks.